CefSharp/ResourceFactoryItem.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+namespace CefSharp
+{
+    public class ResourceFactoryItem
```

Comment:
Renamed class, as it's only used in the context of `DefaultResourceHandlerFactory`.